src/Interpreters/DDLWorker.cpp

```diff
@@ -338,10 +338,10 @@ void DDLWorker::scheduleTasks()
     if (!server_startup)
     {
         /// We will recheck status of last executed tasks. It's useful if main thread was just restarted.
-        auto & min_task = *std::min_element(current_tasks.begin(), current_tasks.end());
+        auto & min_task = current_tasks.front();
```

Comment:
Ok, but let's add an assertion that the first task is the minimum one

Reply:
By entry name over all not completed but started tasks?
Looks complex (tasks to current_tasks always added to the end, so this should not be a problem)

Reply:
Yes, I mean something like
`assert(min_task == *std::min_element(current_tasks.begin(), current_tasks.end()));`

Reply:
But this is not correct, since they will be compared by pointer address, and there is no strict guarantee that the newly allocated pointer will be greater the previous